app/lib/bk/compat/parsers/circleci.rb

```diff
@@ -6,6 +6,7 @@
 require_relative 'circleci/executors'
 require_relative 'circleci/jobs'
 require_relative 'circleci/orbs'
+require_relative 'circleci/orbs/docker'
```

Comment:
this should not be necessary here, it should be included in the `circleci/orbs.rb` file instead